integrations/tensorflow/e2e/keras/keyword_spotting_streaming_test.py

```diff
@@ -0,0 +1,124 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests of streamable Keyword Spotting models implemented in Keras."""
+
+import os
+import sys
+import pathlib
+
+from absl import app
+from absl import flags
+from absl import logging
+import numpy as np
+from pyiree.tf.support import tf_test_utils
+from pyiree.tf.support import tf_utils
+import tensorflow.compat.v2 as tf
+
+# We need to make python aware of the kws_streaming module in third_party, which
+# does not come with bazel BUILD files.
+source_path = str(pathlib.Path(__file__).resolve())
+iree_dir = source_path.replace(
+    "integrations/tensorflow/e2e/keras/keyword_spotting_streaming_test.py", "")
+google_research_dir = os.path.join(iree_dir, 'third_party/google-research')
+sys.path.append(google_research_dir)
```

Comment:
oofdy doofty... Is there a pip for this? Have you taken a look at how this going to work internally? This means the test is basically ignoring its BUILD file dependencies and just mucking about in the file tree :-/

Reply:
Yeah I am definitely not a fan lol... Oleg didn't want me to have to write BUILD files, and this worked in OSS, but I hadn't thought about getting this to work internally 🥴

Unfortunately there isn't a pip for `google-research` or `kws_streaming`. Do you think it would be more effort to create a pip package for `kws_streaming`, or to figure out how to get the BUILD files to open source? At least in the case of the former a pip package would be useful for other people, where as I doubt that anyone else will use the open sourced `bazel` `BUILD` files. IIRC the repo-side setup for a pip package shouldn't be _too_ difficult in this case, but I haven't created any official pip packages before.

Reply:
@stellaraccident is there a better way to do this?